research/notes.org

```diff
@@ -8,67 +7,95 @@ subject are linked.
 :EXPORT_FILE_NAME: spoofax-org-export
 :END:
 #+LATEX_CLASS: article-shifted
-Spoofax is an environment that allows for the completely /declarative/
-definition of a programming language. The definition is done using
+Spoofax is a platform that allows for the completely /declarative/
+definition of a programming language and accompanying IDE
+support\nbsp\cite{Kats10a}. Such a platform is called a /language
+workbench/. The definition of a programming language is done using
 high-level /meta-languages/ for each aspect of the programming
 language.
 
 To define a language declaratively means that one uses the
 meta-languages to specify /what/ the properties of a language are, and
-not /how/ these properties are
-implemented[fn:decl-your-lang:Reference: Declare your language book
-preface]. For example, instead of asking "How do I implement a
-tokenizer and parser for my language?", one asks "What is the syntax
-of my language?". From such a description in a meta-language, the
-tokenizer and parser can be derived, without the designer of the
-language ever having to care about its implementation.
-
-This section goes over the elements that make up the specification of
-a language using Spoofax:
+not /how/ these properties are implemented. For example, instead of
+asking "How do I implement a tokenizer and parser for my language?",
+one asks "What is the syntax of my language?". From such a description
+in a meta-language, the tokenizer and parser can be derived, without
+the designer of the language ever having to care about its
+implementation.
+
+This section goes over the parts that make up the specification of a
+language, and for each part the relevant part of Spoofax is
```

Comment:
`part .. .. part`. Can we maybe do something like `and for each aspect the relevant part/service/something of Spoofax`?